ReleaseNotes.md

```diff
@@ -18,6 +18,72 @@ Release Notes
 OpenRocket 22.02 (2023-02-XX)
 ------------------------
 
+The 22.02 release includes hundreds of new features, bug fixes, and UI improvements, more than we could ever fit into one set of release notes.  These notes summarize the highlights of the entire release; for more detail consult the notes from the five previous public beta releases.
+ 
+Please note that version 22.02 is required for Macs running MacOS 13.0 or later.
```

Comment:
The preferred spelling is "macOS" (I know there are a lot of different ways to write it, I've also used "MacOS" for a long time, but "macOS" is the official spelling).